tasks/php_apt_switchversion.yml

```diff
@@ -16,13 +16,14 @@
       -setup
 
   - name: PHP | Fix php family prefix
-    set_fact: php_family_prefix=php
+    set_fact: php_family_prefix=php7
     when: php_family == "default" and (ansible_distribution_release == "xenial") and  php_present|failed
```

Comment:
and  php_present|failed can be eliminated.

Below there were PPA, that's why we were saving some time on skipping.
while here we set fact, which should be executed always